glide.yaml

```diff
@@ -191,6 +195,12 @@ import:
 # etcd pins a very old version that has contention issues
 - package: github.com/google/btree
   version: master
+# builds
+- package: github.com/projectatomic/buildah
+  repo: https://github.com/nalind/buildah-1.git
+  version: 979+980
+- package: github.com/seccomp/libseccomp-golang
+  version: master
```

Comment:
We don't do this.  We do not depend on a master branch. You may use a semver branch, a tag, or a specific sha.  But not master.